docs/user/environment/2-earth-models/topography.rst

```diff
@@ -0,0 +1,83 @@
+Topography
+==========
+
+We will show how RocketPy interacts with Earth Topography.
+We mainly will use data provided by the NASADEM Merged DEM Global 1 arc second nc.
+
+NASADEM is a digital elevation model based on the Shuttle Radar Topography Mission (SRTM),
+a collaboration between NASA and the National Geospatial-Intelligence Agency (NGA),
+as well as participation from the German and Italian space agencies.
+
+.. seealso::
+
+    You can understand more about NASADEM by reading their documentation at \
+    `NASA's Earth data Search <https://lpdaac.usgs.gov/products/nasadem_hgtv001/>`_.
+
+
+This is a first step forward stopping consider Earth as a constant plane better
+results when we are flying next to mountains or valleys
```

Comment:
The phrasing here is strange, and I think it is missing linking words between "plane" and "better." If you want to keep this phrase, I would rewrite it like

"This is a first step towards stopping considering Earth as a constant plane and having better
results when we are flying next to mountains or valleys."

However, I prefer to directly state the information that topography improves the results when there are mountains and valleys nearby, something along the lines

``` suggestion
Including the topography provides more accurate results when flying next to mountains or valleys.
```